systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaTopicResource.java

```diff
@@ -55,4 +57,18 @@ public static void replaceTopicResource(String resourceName, Consumer<KafkaTopic
     public static void replaceTopicResourceInSpecificNamespace(String resourceName, Consumer<KafkaTopic> editor, String namespaceName) {
         ResourceManager.replaceCrdResource(KafkaTopic.class, KafkaTopicList.class, resourceName, editor, namespaceName);
     }
+
+    public static KafkaTopicList getAllKafkaTopicsWithPrefix(String namespace, String prefix) {
+        KafkaTopicList topics = KafkaTopicResource.kafkaTopicClient().inNamespace(namespace).list();
+        if (prefix != null) {
+            List<KafkaTopic> subsetTopics = new ArrayList<>();
+            for (KafkaTopic kt : topics.getItems()) {
+                if (kt.getMetadata().getName().startsWith(prefix)) {
+                    subsetTopics.add(kt);
+                }
+            }
+            topics.setItems(subsetTopics);
+        }
+        return topics;
+    }
```

Comment:
Isn't this method already in present in `KafkaTopicUtils` ?

Reply:
removed